src/components/TextField/tests/TextField.test.tsx

```diff
@@ -1390,33 +1390,76 @@ describe('<TextField />', () => {
   });
 
   describe('monospaced', () => {
-    it('passes monospaced prop to TextField', () => {
-      const element = mountWithApp(
+    it('applies the monospaced style', () => {
```

Comment:
Interesting, looks like the test wasn't doing anything before, thanks for fixing that! Not sure if there's a lot of value in testing the `className` though since that's an implementation detail of the component.